packages/rocket-uploads3-store-event-aws-infrastructure/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@boostercloud/rocket-uploads3-store-event-aws-infrastructure",
+  "version": "0.9.5699",
```

Comment:
The minor version looks very weird, should be probably according to the Booster version

Reply:
hahaha, what happened here?

Reply:
@NickSeagull @alvaroloes 
hahaha, I've published a LOT of local versions to test the rocket before publushing the PR :P 
So a question for you guys, Does it have to be the newest one? I thought Github action would update this to the new one? Not sure how it works